.github/workflows/snarkjs.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Check if directory path argument is provided
+if [ "$#" -lt 2 ]; then
+    echo "Usage: $0 <directory_path> <curve>"
+    exit 1
+fi
+
+DIR_PATH=$1
+CURVE=$2
+NEW_PTAU=false
```

Comment:
we will need to download the ptau from somewhere, if it is too time consuming to rebuild on the CI.